src/controllers/changeset.controllers.js

```diff
@@ -66,32 +66,21 @@ export const createOrUpdateChangesetFile = async (payload) => {
     );
 
     // Step 2 - Handle "skip" option
-    const changesetFilePath = `${CHANGESET_PATH}/${prNumber}.yml`;
-    const changesetFile = await getFileByPath(
-      headOctokit,
-      headOwner,
-      headRepo,
-      headBranch,
-      changesetFilePath
-    );
+
     if (isSkipEntry(changesetEntriesMap)) {
       await addLabel(baseOctokit, baseOwner, baseRepo, prNumber, SKIP_LABEL);
-      console.log("Skip option found. No changeset file created or updated.");
-      if(changesetFile) {
-        // Delete changeset file if one was previously created
-        const commitMessage = `Changeset file for PR #${prNumber} deleted`;
-        await deleteFileByPath(
-          headOctokit,
-          headOwner,
-          headRepo,
-          headBranch,
-          changesetFilePath,
-          commitMessage,
-          changesetFile.sha
-        );
-      } else {
-        console.log('No changeset file to delete.');
-      }
+
+      // Delete changeset file if one was previously created
+      const commitMessage = `Changeset file for PR #${prNumber} deleted`;
+      await deleteFileByPath(
+        headOctokit,
+        headOwner,
+        headRepo,
+        headBranch,
+        `${CHANGESET_PATH}/${prNumber}.yml`,
```

Comment:
To improve readability, I would recommend initializing a `changesetFilePath` variable toward the top of the `try` block with `${CHANGESET_PATH}/${prNumber}.yml` as the value. This is similar to what we do in the `catch` block below.

Reply:
Can you add that? Thinking you are right.